snippets/matter-diagnostic-logs/boards/nrf54l15dk_nrf54l10_cpuapp.overlay

```diff
@@ -0,0 +1,50 @@
+/* Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/ {
+	cpuapp_sram@2002F340 {
+		compatible = "zephyr,memory-region", "mmio-sram";
+		reg = <0x2002F340 0x1CC0>;
```

Comment:
1CC0=7360, after decreasing it should be 0xCC0

Reply:
Also, shouldn't it be 0x200**3**F340 instead of 0x2002F340?

Reply:
Nvm, it's ok, discussed f2f

Reply:
0xCC0 was indeed incorrect. fixed